charts/rancher-logging/rancher-logging-crd/3.14.0/Chart.yaml

```diff
@@ -2,6 +2,7 @@ annotations:
   catalog.cattle.io/certified: rancher
```

Comment:
@paynejacob why not just delete it and then revert the PR later ? Releasing a chart early is a bit off, there will be confusion on if we can edit it or if we need to make a 3.14.001